astro/src/content/quickstarts/quickstart-ruby-on-rails-api.mdx

```diff
@@ -0,0 +1,351 @@
+* Your fusionAuthBaseUrl is 'http://localhost:9011/'
+
+You can log into the [FusionAuth admin UI](http://localhost:9011/admin) and look around if you want, but with Docker/Kickstart you don't need to.
+
+## Create Your Ruby on Rails Resource Server Application
+
+Now you are going to create a Ruby on Rails API application. While this section builds a simple API, you can use the same configuration to integrate an existing API with FusionAuth.
+
+We are going to be building an API backend for a banking application called ChangeBank. This API will have two endpoints:
+- `make-change`: This endpoint will allow you to call GET with a `total` amount and receive a response indicating how many nickels and pennies are needed to make change. Valid roles are `customer` and `teller`.
+- `panic`: Tellers may call this endpoint to call the police in case of an incident. The only valid role is `teller`.
+
+Both endpoints will be protected such that a valid JSON web token (JWT) will be required in the `Authorization` header in order to be accessed. Additionally, the JWT must have a `roles` claim containing the appropriate role to use the endpoint.
+
+<Aside type="note">
+If you simply want to run the application , there is a completed version in the 'complete-application' directory. You can use the following commands to get it up and running if you do not want to create your own.
+
+```shell
+cd complete-application
+bundle install
+bundle e rails s -p 4001
+```
+
+You can then follow the instructions in the [Run the API](#run-the-api) section started at Get a Token.
+</Aside>
+
+### Initialize The Application
+
+Initialize the Ruby on Rails application using the following:
+
+```
+rails new your-application --api
+cd your-application
+```
+
+Once this is complete you will see a new directory called `your-application` with several sub directories.
+
+You need to update the `Gemfile` to include two new dependencies:
+
+```shell
+gem 'rack-jwt', git: 'https://github.com/FusionAuth/rack-jwt'
+gem 'dotenv-rails'
+```
+
+Your full `Gemfile` will now look like
+
+<RemoteCode url={frontmatter.coderoot + "/complete-application/Gemfile"}
+  lang="text"/>
+
+You may need to modify the version of ruby specified in the Gemfile. As long as Rails 7 is supported, you will be fine.
+
+Then, install these new gems.
+
+```shell
+bundle install
+```
+### Update Routes Config
+
+Update `/config/routes.rb` to include the two new routes that will be created for `/make-change` and `/panic`. Your `routes.rb` file should now match the below code.
+
+<RemoteCode url={frontmatter.coderoot + "/complete-application/config/routes.rb"}
+  lang="ruby"/>
+
+### Add Security
+
+Create a new file to hold your environment variables, directly in the `complete-application` directory called `.env.development`. This new file has two variables that are used to call your FusionAuth instance `FUSIONAUTH_LOCATION` and `CLIENT_ID`, with the values that match below.
```

Comment:
Should this be in the `complete-application` directory or the `your-application` directory.  Assuming it should be the `your-application` directory.